Tests/ASNavigationControllerTests.mm

```diff
@@ -1,5 +1,5 @@
 //
-//  ASNavigationControllerTests.mm
```

Comment:
Should probably rename the file too.

Reply:
Agreed! @zboralski could you add this rename and we can get this merged?